paddle/testing/paddle_gtest_main.cc

```diff
@@ -66,6 +66,7 @@ int main(int argc, char** argv) {
 #if defined(PADDLE_WITH_ASCEND_CL)
   envs.push_back("selected_npus");
   envs.push_back("npu_config_path");
+  envs.push_back("npu_precision_mod");
```

Comment:
No need to add this, the implementation has been changed, you can pull the latest paddle.

Reply:
done. thanks.